src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarLoreUtil.java

```diff
@@ -0,0 +1,364 @@
+        if (points == null) return getFormattedUnknown();
+
+        return colorValue(String.valueOf(points));
+    }
+
+    public static String getFormattedDefenderPoints(PersistentDataContainer container) {
+        Integer points = getSiegeDefenderPoints(container);
+        if (points == null) return getFormattedUnknown();
+
+        return colorValue(String.valueOf(points));
+    }
+
+    public static String getFormattedWinner(PersistentDataContainer container) {
+        String winning_side = getSiegeWinningSide(container);
+        if (winning_side == null) return getFormattedUnknown();
+
+        switch (winning_side) {
+            case "ATTACKERS":
+                return getFormattedAttacker(container);
+            case "DEFENDERS":
+                return getFormattedDefender(container);
+            case "NOBODY":
+                return colorValue(SiegeSide.NOBODY.name());
+            default:
+                throw new RuntimeException("Unknown winning SiegeSide in SiegeWar lore item.");
+        }
+    }
+
+    public static String getFormattedOpposition(PersistentDataContainer container) {
+        String winning_side = getSiegeWinningSide(container);
+        if (winning_side == null) return getFormattedUnknown();
+
+        switch (winning_side) {
+            case "ATTACKERS":
+                return getFormattedDefender(container);
+            case "DEFENDERS":
+                return getFormattedAttacker(container);
+            case "NOBODY":
+                return colorValue(SiegeSide.NOBODY.name());
+            default:
+                throw new RuntimeException("Unknown winning SiegeSide in SiegeWar lore item.");
+        }
+    }
+
+    public static String getFormattedStatus(PersistentDataContainer container) {
+        String status = getSiegeStatus(container);
+        if (status == null) return getFormattedUnknown();
+
+        return colorValue(status);
+    }
+
+    public static String getFormattedStart(PersistentDataContainer container) {
+        Long time = getSiegeStart(container);
+        if (time == null) return getFormattedUnknown();
+
+        return colorValue(new SimpleDateFormat(Translation.of("siege_lore_date_format")).format(time));
+
+    }
+
+    public static String getFormattedEnd(PersistentDataContainer container) {
+        Long time = getSiegeEnd(container);
+        if (time == null) return getFormattedUnknown();
+
+        return colorValue(new SimpleDateFormat(Translation.of("siege_lore_date_format")).format(time));
+    }
+
+    public static void shieldItem(ItemMeta meta, PersistentDataContainer data) {
+        if (!hasLoreKey(data)) return;
+
+        String name = Translation.of("siege_lore_two_values",
+                getFormattedTown(data),
+                colorNeutral(Translation.of("siege_lore_shield_name")));
+
+        meta.setDisplayName(name);
+
+        bannerLore(meta, data);
+    }
+
+    public static String bannerChat(PersistentDataContainer data) {
```

Comment:
This needs a better method name so someone knows what it does just reading the method.